app/auto-eligibility/register-your-interest/process-eligible-customer.js

```diff
@@ -0,0 +1,32 @@
+const emailNotifier = require('../email-notifier')
+const customerDbTable = require('../db/customer.db.table')
+
+const processEligibleCustomer = async (customer) => {
+  console.log(`Processing eligible customer: ${JSON.stringify({
+     ...customer
+  })}`)
+  if (customer.sbiAlreadyRegistered()) {
+    console.log('The customer`s sbi has already been registered')
+    return await emailNotifier.sendIneligibleApplicationEmail(
+      customer.sbi,
+      customer.crn,
+      customer.businessEmail
+    )
+  }
+  if (customer.businessEmailHasMultipleDistinctSbi()) {
+    console.log('The customer`s business email has multiple sbi which as of now the system does not support')
+    return await emailNotifier.sendIneligibleApplicationEmail(
+      customer.sbi,
+      customer.crn,
+      customer.businessEmail
+    )
+  }
+  if (customer.alreadyOnWaitingList()) {

```

Comment:
I think this is caught by the sbiAlreadyRegistered() check

Reply:
done - removed